docs/docstring_template.md

```diff
@@ -0,0 +1,72 @@
+# Docstring Template
+--------------------
+
+Below is a docstring example. It contains a suggested docstring layout, example text and example MathJax.
+
+The layout below will render properly in Read the Docs. Please adjust the example text and MathJax as required.
+
+Note: backticks render differently in docstrings to elsewhere. Should you wish to include text in backticks, please use double backticks - e.g. ``scores.continuous.mean_error``. (If you only use one set of backticks, the text will instead be rendered as italics in Read the Docs). 
```

Comment:
With the `scores.continuous.mean_error` example, when writing the name of a function it is better to use ":py:func:`"

Consider updating this to suggest writing function names with ":py:func:"  and non-functions that you want to render as a block (or whatever it's called) with double backtracks (e.g., `reduce_dims="all"`). I hope that makes sense

Reply:
I see that you use ":py:func:" further down in the example. Consider just updating this example to something like `reduce_dims="all"`